caliban/docker/push.py

```diff
@@ -31,21 +36,47 @@ def _image_tag_for_project(project_id: str, image_id: str) -> str:
 
   """
   project_s = project_id.replace(":", "/")
-  return "gcr.io/{}/{}:latest".format(project_s, image_id)
+  base = f"gcr.io/{project_s}/{image_id}"
+  return f"{base}:latest" if include_tag else base
+
+
+def _gcr_list_tags(project_id: str, image_id: str):
+  """Returns a sequence of metadata for all tags of the supplied image_id in the
+  supplied project.
+
+  """
+  image_tag = _image_tag_for_project(project_id, image_id, include_tag=False)
+  cmd = [
```

Comment:
should we pass the `--project` flag here to the gcloud command, as we have the `project_id` handy?

Reply:
NIce!